cmd/brew-bootstrap-nodenv-node

```diff
@@ -44,7 +44,7 @@ if ! nodenv version-name &>/dev/null; then
 fi
 
 if [ "$(nodenv exec node --version)" != "$(node --version)" ]; then
-  abort 'You need to add `eval "$(nodenv init -)"` to your .bash_profile/.bashrc/.zshrc!'
+  abort 'Error: add `eval "$(nodenv init -)"` to the end of your .bash_profile/.zshrc!'
```

Comment:
Mabye we could tell to run: ` echo 'eval "$(nodenv init -)"' >> ~/.bash_profile/.zshrc!`